app/controllers/images_controller.rb

```diff
@@ -24,9 +24,9 @@ def fetch
   # assign attributes directly after rails update
   def create
     @warpable = Warpable.new
+    @warpable.history = 'None'
```

Comment:
What's the purpose of this attribute?

Reply:
This was just done to remove the MySQL2 error that says that ```history``` doesn't have any default value.